### ISSUE TITLE
Courses being duplicated in emails

### ISSUE THREAD

**Issue by j-jasik** [betheluniversity/sciencelabs-python#176]
In the email that gets sent out after a session is done I've noticed that sometimes courses get duplicated (see attached photo for example). I've specifically noticed it happening for math lab. I'm not sure if it's showing up like that for the student and their selecting both, or if somewhere on the backend there's some weird logic happening. Either way, it should be looked into.
<img width="745" alt="Screen Shot 2019-10-21 at 9 01 09 AM" src="https://user-images.githubusercontent.com/36043372/67212210-a0a6fc00-f3e1-11e9-9e0a-2156e4e7f5a7.png">

**Comment by bam95899** [comment]
I dug into this a little and it is definitely an anomaly. From my research I believe the duplication of courses occurs because two student sessions are being created somehow, but I have yet to figure out how. I have confirmed that when duplicate courses occur in the same session for a student there are also duplicated student sessions.

Examples of this happening: 
https://tutorlabs.bethel.edu/mathlab/report/student/41233
https://tutorlabs.bethel.edu/mathlab/report/student/41571

**Comment by j-jasik** [comment]
Yeah I've looked into it a little bit a couple of times and I don't know what's going on there... I wonder if it's a side effect of signing in/out of CAS so much for the open sessions? If that's the case I don't know what we can do about it. I've also wondered if there's some sort of redirect loop that can get hit sometimes so it calls the method that creates the student session more than once? It's a mystery.
